frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -338,56 +404,152 @@ export default function AnalysisParamForm({
-                value={!checked ? "" : inputVals.endTime}
-                onChangeFn={(e) => {
-                  updateParams({
-                    endTime: e.target.value,
-                  });
-                }}
-              >
-                <ErrorText id="endTimeError" role="errorMsg">
-                  {errorMessages.endTime}
-                </ErrorText>
-              </FormInput>
-            </InputErrorContainer>
-          </ParamContainer>
+          <WAOverlayContainer>
+            {checkedWindow || <WAOverlay />}
+            <ParamContainer>
+              <Label htmlFor="startTime">Start Time (s):</Label>
+              <InputErrorContainer>
+                <FormInput
+                  name="startTime"
+                  placeholder={checkedWindow ? "0" : ""}
+                  value={!checkedWindow ? "" : inputVals.startTime}
+                  onChangeFn={(e) => {
+                    updateParams({
+                      startTime: e.target.value,
+                    });
+                  }}
+                >
+                  <ErrorText id="startTimeError" role="errorMsg">
+                    {errorMessages.startTime}
+                  </ErrorText>
+                </FormInput>
+              </InputErrorContainer>
+            </ParamContainer>
+            <ParamContainer>
+              <Label htmlFor="endTime">End Time (s):</Label>
+              <InputErrorContainer>
+                <FormInput
+                  name="endTime"
+                  placeholder={checkedWindow ? "(End of recording)" : ""}
+                  value={!checkedWindow ? "" : inputVals.endTime}
+                  onChangeFn={(e) => {
+                    updateParams({
+                      endTime: e.target.value,
+                    });
+                  }}
+                >
+                  <ErrorText id="endTimeError" role="errorMsg">
+                    {errorMessages.endTime}
+                  </ErrorText>
+                </FormInput>
+              </InputErrorContainer>
+            </ParamContainer>
+          </WAOverlayContainer>
         </WindowAnalysisContainer>
+        <WALabel>
+          <CheckboxWidget
+            color={"secondary"}
+            size={"small"}
+            handleCheckbox={(checkedAdvanced) => setCheckedAdvanced(checkedAdvanced)}
+            checkedState={checkedAdvanced}
+          />
+          Use Advanced Analysis
+        </WALabel>
+        <AdvancedAnalysisContainer>
+          <WAOverlayContainer>
+            {checkedAdvanced || <WAOverlay />}
+            <TwoParamContainer>
+              <Label htmlFor="prominenceFactorMin">Prominence (uN):</Label>
+              <InputErrorContainer>
+                <label htmlFor="prominenceFactorMin">min</label>
+                <FormModify>
+                  <FormInput
+                    name="prominenceFactorMin"
+                    placeholder={checkedAdvanced ? "0" : ""}
```

Comment:
and I would make the placeholder the default values that get used in Pulse3D

Reply:
I agree, and depending on what our default values are we'll need to handle the max/min combos carefully. For both prominence and width, scipy will let us enter either None, a min only, or a max and min. We can't enter just a max which is unfortunate. So for example, one case that's tricky to decide what to do is if the user enters only a max and the value they choose is lower than the default min. We should probably discuss this more in standup or set some time aside for it. Might also be good to see what Hamed thinks